axelrod/strategies/axelrod_second.py

```diff
@@ -199,3 +202,194 @@ def strategy(self, opponent: Player) -> Action:
+            Set FD = 0                                                                                                                                                                                                                                                  
+            Set the variable 'NO':                                                                                                                                                                                                                                      
+            NO = ((NO * NK) + 3 - (3 * J) + (2 * JA) - (JA * J) / (NK + 1)                                                                                                                                                                                              
+            Where JA and J are the last moves of the strategy and the opponent (C=0, D=1)                                                                                                                                                                       
+            Increase the value of NK by 1                                                                                                                                                                                                                               
+            Cooperate                                                                                                                                                                                                                                                   
+                                                                                                                                                                                                                                                                        
+    Tranquilizer came in 27th place in Axelrod's second torunament. 
+   
+
+    Names:
+
+    - Tranquilizer: [Axelrod1980]_
+    """
+
+    name = 'Tranquilizer'
+    classifier = {
+        'memory_depth': float('inf'),
+        'stochastic': True,
+        'makes_use_of': {"game"},
+        'long_run_time': False,
+        'inspects_source': False,
+        'manipulates_source': False,
+        'manipulates_state': False
+    }
+
+    def __init__(self):
+        super().__init__()
+        self.num_turns_after_good_defection = 0 # equal to FD variable in Fortran code
+        self.opponent_consecutive_defections = 0
+        self.one_turn_after_good_defection_ratio= 5
+        self.two_turns_after_good_defection_ratio= 0
+        self.one_turn_after_good_defection_ratio_count = 1
+        self.two_turns_after_good_defection_ratio_count = 1
+        self.current_score = 0
+        self.dict = {C: 0, D: 1}
+
+
+    def update_cooperative_state(self, opponent):  
```

Comment:
This now updates more than the cooperative state (it also updates the consecutive defections) so let's go back to calling it:

```
def update_state
```